CDCgov/prime-reportstream
# Add TI receiver API endpoint details to Flexion org settings for results

This PR adds REST transport receiver settings for the TI results use case as described in [TI's issue #615](https://github.com/CDCgov/trusted-intermediary/issues/615)

Test Steps (local):
1. Run `./prime multiple-settings set --input ./settings/staging/0166-flexion-staging-results-handling.yml`
2. Send an ORU message to the waters api with the `flexion.simulated-lab` client (use [004_MN_ORU_R01_NBS.hl7](https://github.com/CDCgov/trusted-intermediary/blob/main/examples/MN/004_MN_ORU_R01_NBS.hl7) if you need sample ORU)
    `curl --header 'Content-Type: application/hl7-v2' --header 'Client: flexion.simulated-lab' --header 'Authorization: Bearer token' --data-binary '@ORU_R01_MN_natus.hl7' 'http://localhost:7071/api/waters'`
4. Verify the request went through without any errors and sent to TI's results API endpoint 

## Checklist

### Testing
- [ ] Ran `./gradlew testSmoke`
- [ ] Ran `./gradlew testIntegration`
